.github/workflows/tests.yml

```
@@ -20,6 +22,10 @@ jobs:
     strategy:
       matrix:
         php-versions: ['7.3', '7.4', '8.0', '8.1', '8.2', '8.3', '8.4']
```

You need to remove this 8.4 here, when including it via the below:
```suggestion
        php-versions: ['7.3', '7.4', '8.0', '8.1', '8.2', '8.3']
```

See also https://github.com/PrivateBin/PrivateBin/pull/1300/files

---

Oh just saw https://github.com/PrivateBin/PrivateBin/pull/1300 you did the same, damn…

---

SHould not be needed but yeah if it works.